modules/labs-react/tabs/lib/cursor/hooks.tsx

```diff
@@ -69,3 +73,50 @@ export const useRovingFocus = ({state, events}: CursorModel, elemProps = {}) =>
     elemProps
   );
 };
+
+/**
+ * Handles the roving focus behavior of a Cursor model. It should be added to the element
+ * representing a list item. It does not handle the tabIndex.
+ *
+ * @see https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex
+ */
+export const useRovingFocusItem = ({state}: CursorModel, elemProps = {}) => {
```

Comment:
@alanbsmith This hook is how focus management is working on deleted list items. No subscriptions needed.